website/blog/2018-08-27-7.0.0.md

```diff
@@ -0,0 +1,462 @@
+
+- Drop support for un-maintained Node versions: 0.10, 0.12, 4, 5 ([details](http://babeljs.io/blog/2017/09/12/planning-for-7.0#drop-support-for-unmaintained-node-versions-010-012-5-4315-https-githubcom-babel-babel-issues-4315))
+- Move us to the `@babel` namespace by switching to using "scoped" packages ([details](http://babeljs.io/blog/2017/12/27/nearing-the-7.0-release#renames-scoped-packages-babel-x)). This helps differentiate official packages, so `babel-core` becomes `@babel/core` (and no squatting)
+- Remove (and stop publishing) any yearly presets (`preset-es2015`, etc) ([details](http://babeljs.io/blog/2017/12/27/nearing-the-7.0-release#deprecated-yearly-presets-eg-babel-preset-es20xx)). [`@babel/preset-env`](https://babeljs.io/docs/en/next/babel-preset-env.html) replaces the need for these as it includes all yearly additions as well as the ability to target a specific set of browsers
+- Also drop the ["Stage"](https://tc39.github.io/process-document/) presets (`@babel/preset-stage-0`, etc) in favor of opting into individual proposals. Similarly remove proposals from `@babel/polyfill` by default ([details](https://github.com/babel/babel/pull/8440)). Please consider reading the whole [post](https://babeljs.io/blog/2018/07/27/removing-babels-stage-presets) on this for more explanation.
+- Some packages have renames: any TC39 proposal plugin will now be `-proposal` instead of `-transform` ([details](http://babeljs.io/blog/2017/12/27/nearing-the-7.0-release#renames-proposal)). So `@babel/plugin-transform-class-properties` becomes `@babel/plugin-proposal-class-properties`.
+- Introduce a `peerDependency` on `@babel/core` for certain user-facing packages (e.g. `babel-loader`, `@babel/cli`, etc) ([details](http://babeljs.io/blog/2017/12/27/nearing-the-7.0-release#peer-dependencies-integrations))
+
+## JavaScript Config Files
+
+We are introducing [`babel.config.js`](https://babeljs.io/docs/en/next/babelconfigjs.html).
+
+`*.js` configuration files are fairly common in the JavaScript ecosystem. ESLint and Webpack both allow for `.eslintrc.js` and `webpack.config.js` configuration files, respectively.
+
+Note that this doesn't mean that it replaces writing a config with `.babelrc` or in `package.json`. It isn't a requirement, but having this may be useful in certain cases. Below is the case of only compiling with a plugin in "production".
+
+```js
+var env = process.env.NODE_ENV;
+module.exports = {
+  plugins: [
+    env === "production" && "babel-plugin-that-is-cool"
+  ].filter(Boolean)
+};
+```
+
+`babel.config.js` has a different config resolution than a `.babelrc`. It will always resolve everything from that file vs. originally when Babel would do a lookup from each file upward until it found a config. This makes it possible to take advantage of the next feature posted below, `overrides`.
```

Comment:
I may be reading this wrong. For me it looks like it's saying:

"For `babel.config.js` we read everything from the file, for `.babelrc` we resolve up until we find a file"

It seems like the first part is talking about how a file is read and the second part describes how a file is resolved.